src/main/java/uk/gov/companieshouse/officerfiling/api/controller/OfficerFilingControllerImpl.java

```diff
@@ -82,9 +83,9 @@ public OfficerFilingControllerImpl(final TransactionService transactionService,
     public ResponseEntity<Object> createFiling(@PathVariable final String transId,
             @RequestBody @Valid @NotNull final OfficerFilingDto dto,
             final BindingResult bindingResult, final HttpServletRequest request) {
-        final var logMap = LogHelper.createLogMap(transId);
-
-        logger.debugRequest(request, "POST", logMap);
+        logger.debugContext(transId, "POST", new LogHelper.Builder(transId)
+                .withRequest(request)
+                .build());
```

Comment:
I would advise using the RequestLogger in the structured logging library for this log.

Reply:
See above comment.